### DISCUSSION TITLE
filter sharepoint list column keywords.

### DISCUSSION THREAD

**Discussion by j0tas2002** [microsoft-search/pnp-modern-search#2770] [General]
I am trying to create a query to filter keywords from a SharePoint list on PnP modern search web part. I am using the query below and doesn't seem to return any results.

 {searchTerms} (https://tenant.test.sharepoint.com/site/234::{/list//} AND Title:({searchTerms}) OR Animal({searchTerms})

`path:https://contoso.sharepoint.com/sites/site/Lists/MyList` + the contenttype. If there are columns you don't want to be searchable you can set the column to be excluded from indexing. Programatically there is a property on a field named `NoCrawl` which you can set.

You can also target columns specifically with:

`Title:({searchTerms}) OR CaseNumber:({searchTerms})`

_Originally posted by @wobba in https://github.com/microsoft-search/pnp-modern-search/issues/508#issuecomment-703262037_

**Comment by kasperbolarsen** [comment]
I would recommend using the SharePoint search query tool to debug the query,  https://github.com/SharePoint/PnP-Tools/tree/master/Solutions%2FSharePoint.Search.QueryTool

I guess "Animal" must be an alias, right?
